packages/@aws-cdk/aws-codebuild/lib/project-logs.ts

```diff
@@ -0,0 +1,69 @@
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+
+/**
+ * Information about logs built to an S3 bucket for a build project.
+ */
+export interface S3LogsConfig {
+  /**
+   * Encrypt the S3 build log output
+   * @default true
+   */
+  readonly encrypted?: boolean;
+
+  /**
+   * The S3 Bucket to send logs to
+   */
+  readonly bucket: s3.IBucket;
+
+  /**
+   * The path prefix for S3 logs
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+
+  /**
+   * The current status of the logs in Amazon CloudWatch Logs for a build project
+   * @default true
+   */
+  readonly enabled?: boolean;
+}
+
+/**
+ * Information about logs built to a Cloudwatch Log Group for a build project.
+ */
+export interface CloudwatchLogsConfig {
+  /**
+   * The Log Group to send logs to
+   */
+  readonly logGroup: logs.ILogGroup;
```

Comment:
I wonder whether it makes sense to make `logGroup` optional. You said one of the reasons for having the `enabled` property is to disable the default logging configuration. I think making `logGroup` required makes that use case pretty difficult to accomplish...?

We can always add runtime validation that `logGroup` needs to be specified if `enabled` is true.

Reply:
Great point!